File: CS/CS.NET-VB.NET/FormToForm/VB2008FormToForm/VB2008FormToForm/CustomEventArgs.vb
' delegate void CustomEventHandler(object sender, CustomEventArgs e);
Delegate Sub CustomEventHandler(ByVal sender As System.Object, ByVal e As CustomEventArgs)

Class CustomEventArgs
    Inherits EventArgs

    Private TextBoxText As String

    Public Sub New(ByVal TextBoxText As String)
        Me.TextBoxText = TextBoxText
    End Sub

    Friend ReadOnly Property TextBoxTextChanged() As String
        Get
            Return TextBoxText
        End Get
    End Property

End Class

File: CS/CS.NET/Windows Forms .NET/FormToForm/EventArgs/CS2008FormToForm/CS2008FormToForm/CustomEventArgs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


delegate void CustomEventHandler(object sender, CustomEventArgs e);

class CustomEventArgs : EventArgs 
{
    private string TextBoxText;

    public CustomEventArgs(string TextBoxText)
    {
        this.TextBoxText = TextBoxText;
    }

    internal string TextBoxTextChanged
    {
        get
        {
            return TextBoxText;
        }
    }
}

